pkg/apis/core/validation/validation.go

```diff
@@ -352,6 +352,14 @@ func ValidateImmutableField(newVal, oldVal interface{}, fldPath *field.Path) fie
 	return apimachineryvalidation.ValidateImmutableField(newVal, oldVal, fldPath)
 }
 
+func ValidateImmutableLabel(newVal string, oldVal string, label string, fldPath *field.Path) field.ErrorList {
```

Comment:
It looks like there is no unit test for this function.
In k/k repo, do we need to add a unit test against such a function?

Reply:
It makes sense. Higher-level unit tests would be helpful rather than tests only for the lowest level.

Reply:
Marking as resolved